### PR TITLE
Add Rakuten Mobile

### PR DESCRIPTION
**Pull request by ghost**
Could you please prepare an image for me?
Icon source
- https://pbs.twimg.com/profile_images/1234714203627220993/EjUYzXJb_400x400.jpg
- https://network.mobile.rakuten.co.jp/google-search-icon.png

### CODE REVIEW

**Comment by Carlgo11** [comment]
And it took me 20 seconds to find :stuck_out_tongue_closed_eyes: 

https://user-images.githubusercontent.com/3535780/154981403-7b8b1933-a228-4cc3-8a02-883c24e004da.mp4

**Comment by phallobst** [comment]
That logo does not show up for me in the Google image results within Chrome/Opera, but does in Firefox. :man_shrugging:

**Comment by ghost** [comment]
> And it took me 20 seconds to find 😝

I didn't know that Google Image Search allows you to specify svg. I've never heard of it before.
Thank you for the instruction!

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
`entries/n/network.mobile.rakuten.co.jp` is missing the JSON file extension.

**Review by Carlgo11 — APPROVED** [review]

File: entries/n/network.mobile.rakuten.co.jp.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Rakuten Mobile": {
+    "domain": "network.mobile.rakuten.co.jp",
+    "contact": {
+      "facebook": "RakutenMobile",
+      "twitter": "Rmobile_Support"
```
Comment (phallobst) [inline entries/n/network.mobile.rakuten.co.jp.json]:
```suggestion
      "twitter": "Rmobile_Support",
      "language": "ja"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**